cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -750,5 +751,66 @@ TEST(Substrait, ExtensionSetFromPlanMissingFunc) {
           &ext_set));
 }
 
+Result<std::string> GetSubstraitJSON() {
+  ARROW_ASSIGN_OR_RAISE(std::string dir_string,
+                        arrow::internal::GetEnvVar("PARQUET_TEST_DATA"));
+  auto file_name =
+      arrow::internal::PlatformFilename::FromString(dir_string)->Join("binary.parquet");
+  auto file_path = file_name->ToString();
+  std::string substrait_json = R"({
+    "relations": [
+      {"rel": {
+        "read": {
+          "base_schema": {
+            "struct": {
+              "types": [ 
+                         {"binary": {}}
+                       ]
+            },
+            "names": [
+                      "foo"
+                      ]
+          },
+          "local_files": {
+            "items": [
+              {
+                "uri_file": "FILENAME_PLACEHOLDER",
+                "format": "FILE_FORMAT_PARQUET"
+              }
+            ]
+          }
+        }
+      }}
+    ]
+  })";
+#ifdef _WIN32
+  // Path is supposed to start with "X:/..."
+  file_path = "file:///" + file_path;
+#else
+  // Path is supposed to start with "/..."
+  file_path = "file://" + file_path;
+#endif
+  std::cout << "File Path : >>>>" << file_path << std::endl;
```

Comment:
Remove this print?

Reply:
Yes of course, still testing the windows CI breaks. Will remove it after the fix.